src/System.Management.Automation/engine/Interop/Windows/WNetGetConnection.cs

```diff
@@ -52,26 +49,26 @@ internal static int GetUNCForNetworkDrive(char drive, out string? uncPath)
 
             if (errorCode == ERROR_SUCCESS)
             {
-                uncPath = uncBuffer.Slice((int)bufferSize).ToString();
+                uncPath = uncBuffer.Slice(0, (int)bufferSize).ToString();
```

Comment:
In debugger I see extra `\0` last char in `uncPath`. If you see the same we should (below too) something like:
```suggestion
                uncPath = uncBuffer.Slice(0, (int)(bufferSize - 1)).ToString();
```

Reply:
I thought that `ToString()` would handle the null terminator, but looking at the dotnet/runtime code, it'll happily keep the null terminator.  Fixed.